packages/ui-components/src/components/select-option/select-option.types.ts

```diff
@@ -11,12 +17,16 @@ export interface ISelectOption {
 	disabled?: boolean;
 	/** (optional) If `true` the item is selected */
 	selected?: boolean;
-	/** (optional) If `true` the item is in an indeterminate state (partially selected) */
-	indeterminate?: boolean;
 	/** (optional) If `true` the item is togglable */
 	togglable?: boolean;
-	/** (optional) If `true` styles the item to fit content below label */
-	hasBottomSlot?: boolean;
+	/** (optional) If `true` the item is only for presenting and cannot be selected. */
+	presenter?: boolean;
```

Comment:
Presenter doesn't seem like a good property name, I not something like selectable?

Reply:
Selectable does seems clearer, will rename it.